FND/Places/Countries.rdf

```diff
@@ -102,11 +102,56 @@
 		<skos:definition>community in which people live or have lived, without being specific as to size, population or importance</skos:definition>
 	</owl:Class>
 	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-plc-cty;hasCity">
+		<rdfs:label>has city</rdfs:label>
+		<skos:definition>indicates the name of a city</skos:definition>
```

Comment:
But it might not be part of an address - and I agree that it is circular, so perhaps it needs some improvement.  If it is only useful as a part of an address then ok, but I think it could be used more broadly.

Reply:
Changed the definition of hasCity to 'indicates the name of a large, permanent, and densely settled place", added an explanatory note about typical size, and extended the usage note to say that for the purposes of an address, this property can be used to represent any village, town, or city.